packages/tabs/src/component-tabs.tsx

```diff
@@ -146,9 +146,9 @@ export const Tabs = (props: TabsProps) => {
         onKeyDown={handleKeyDown}
       >
         {Children.map(children, (child: any) => {
-          return cloneElement(child, {
-            setActive: change,
-            isActive: child.props.name === active,
+          return child && cloneElement(child, {
+                setActive: change,
+                isActive: child?.props?.name === active,
```

Comment:
Is it tested for the case when the child is just a regular element like `<div/>` ?
Otherwise it will output such error:
```
React does not recognize the `isActive` prop on a DOM element.
If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `isactive` instead.
If you accidentally passed it from a parent component, remove it from the DOM element.
```

Reply:
Code|Output
---|---
![image](https://github.com/warp-ds/react/assets/37986637/8d1e3da4-17ce-43db-9ed0-96fc84e0ecc6)|<img width="627" alt="image" src="https://github.com/warp-ds/react/assets/37986637/2ed6c1c5-6dd2-4935-beab-0553875f85e7">